### DISCUSSION TITLE
HTML學習網站-W3 schools

### DISCUSSION THREAD

**Discussion by dabbie109303501** [1101-GS4719-C/Python-Community-GS4719-C#880] [General]
上課時介紹到的html讓你心癢癢，也想寫一個屬於自己的網站嗎?
W3 schools裡面有介紹很多上課沒有教的語法
也有範例可以操作
![image](https://user-images.githubusercontent.com/75252877/205982717-f98e8e70-7408-4fe3-b65d-023142901d3a.png)
而且除了html 也有很多其他的程式語言的語法可以看
推薦給大家~
https://www.w3schools.com/html/default.asp

**Comment by Chiway1229** [comment]
太感謝推薦教學網站了
我的CSS終於有救了

**Comment by a45320614** [comment]
這個網站太方便了吧，除了html也有其他程式相關的資源可以學習~

**Comment by Rodrigo-dev-bit** [comment]
有時候忘了怎麼寫基本的語法我就會去這個網站複習一下，很棒。

**Comment by dinosaur1009** [comment]
這個網站除了HTML還有其他不同程式語言 超級方便ㄉ

**Comment by ching020721** [comment]
範例、練習、測驗都有，真的能快速學習到欸很方便

**Comment by StevenWanglolz** [comment]
真的各種程式語言都有耶，現在線上自主學習的管道真的好多!

**Comment by Kotori0501** [comment]
有了這個網站，說不定又可以讓程式更加進步
製作出屬於自己的網站
